weaviate/collections/classes/filters.py

```diff
@@ -461,7 +461,7 @@ def greater_than(self, count: int) -> _FilterValue:
             operator=_Operator.GREATER_THAN,
         )
 
-    def greater_or_equal(self, count: int) -> _FilterValue:
+    def greater_or_equal(self, count: int) -> _Filters:
         """Filter on whether the number of references is equal to the given integer.
 
         Arguments:
```

Comment:
Nice! Could you add a test with `&=` and `/=` to make sure that they work without warning? Can just be a unit test in test/collection/test_filter.py